src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -85,24 +86,24 @@ public void updateTaskForStatus(Protos.TaskStatus status) {
       // set the status to initialized
       if (status.getMessage().equals(HDFSConstants.NAME_NODE_INIT_MESSAGE)
           || (currentAcquisitionPhase.equals(AcquisitionPhase.RECONCILING_TASKS)
-              && !isNameNode1Initialized())
+          && !isNameNode1Initialized())
           || (status.getMessage().equals(HDFSConstants.NAME_NODE_BOOTSTRAP_MESSAGE)
-              && !isNameNode1Initialized())) {
+          && !isNameNode1Initialized())) {
         nameNode1TaskMap.clear();
         nameNode1TaskMap.put(status, true);
       } // If bootstrapping the second NN or reconciling the second NN,
-        // set the status to initialized
+      // set the status to initialized
       else if ((status.getMessage().equals(HDFSConstants.NAME_NODE_BOOTSTRAP_MESSAGE)
           && !isNameNode2Initialized())
           || (currentAcquisitionPhase.equals(AcquisitionPhase.RECONCILING_TASKS)
-              && !isNameNode2Initialized())) {
+          && !isNameNode2Initialized())) {
         nameNode2TaskMap.clear();
         nameNode2TaskMap.put(status, true);
       } // If the first NN is not running, set the status to running
-       else if (nameNode1TaskMap.isEmpty()) {
+      else if (nameNode1TaskMap.isEmpty()) {
         nameNode1TaskMap.put(status, false);
       } // If the second NN is not running, set the status to running
```

Comment:
Nit: I dislike this weird comment style. Can we just put them inside the scope they describe?
```
} else if (foo && bar) {
    // Foo exists and bar is set, so do xyz.
    ...
} else if (!baz) {
    // Baz is not running, set the staus to running.
    ...
}
```